ibazel/ibazel.go

```diff
@@ -57,16 +60,20 @@ type IBazel struct {
 	cmd       command.Command
 	args      []string
 	bazelArgs []string
+	noLiveReload bool
```

Comment:
Could we make the variable "liveReload" and then set it to true by default? Having negations in variable names makes my tiny monkey brain hurt.

Reply:
The command line flag was chosen to match the iblaze `nolive_reload` so it would be consistent between the two. Variable here was named to match the flag. Are you thinking change the variable here to `liveReload` and be true by default and keep the flag as `nolive_reload` to be consistent with iBlaze? I like that variable naming better. Agree that negated variables are harder to read.